client/components/dashboard/dashboard-toolbar-directive.js

```diff
@@ -99,7 +107,7 @@ goog.scope(function() {
          * @export
          */
         this.downloadDashboard = function() {
-          let selectedDashboard = this.dashboardSvc.current.model;
+          var selectedDashboard = this.dashboardSvc.current.model;
```

Comment:
I had put them in a test (which still run ES5), and apparently got overzealous in my cleanup.